test/sdk-benchmarks/README.md

```diff
@@ -3,27 +3,31 @@
 
 This module contains sdk benchmark harness using [JMH].
 
-There are three ways you can run benchmarks.
+There are three ways to run benchmarks.
 
-- From IDE
-  
-  You can run the main method within each Benchmark class from your IDE. Note that you might need to
-  adjust the jmh annotations based on your test environment.
-  
-- From Maven command
+- Using the executable JAR (Preferred Usage per JMH site)
 ```bash
-   mvn install -P quick -pl :sdk-benchmarks --am
-   mvn install -pl :bom-internal
-   cd test/sdk-benchmarks
-   mvn exec:exec
-```   
+mvn clean install -P quick -pl :sdk-benchmarks --am
 
-- From command line  
-```bash
-mvn install -P quick -pl :sdk-benchmarks --am
+# Run specific benchmark
+java -jar target/benchmarks.jar ApacheHttpClientBenchmark
 
-# Run 3 warm up iterations, 3 benchmark iterations, 1 fork
+# Run all benchmarks: 3 warm up iterations, 3 benchmark iterations, 1 fork
 java -jar target/benchmarks.jar -wi 3 -i 3 -f 1
 ```
 
+- Using`mvn exec:exec` commands to invoke `BenchmarkRunner` main method
+```bash
+   mvn clean install -P quick -pl :sdk-benchmarks --am
+   mvn clean install -pl :bom-internal
+   cd test/sdk-benchmarks
+   mvn exec:exec
+```   
+
+- From IDE
```

Comment:
Since running from the IDE may be slower, let's add a note about that as well.

Reply:
Yeah, added more notes in README